### DISCUSSION TITLE
How much package loss is ok in LAN UDP

### DISCUSSION THREAD

**Discussion by DaReaLDeviL** [esnet/iperf#1473] [Q&A]
Hello, I just started to messaure a LAN:

[Windows 11 PC]---[Switch]---1Gbit copper---[Switch]---10Gbit---[Switch]---[Windows 10 PC]

Now I trie different types of udp traffic and I always end ab at about 0.1-0.15% on packet loss. Is this normal or do I have to check something?

Thank you in advanced.

**Comment by davidBar-On** [comment]
Hi, few questions:
1. Can you give more details about the "different types of udp traffic" you used?  Especially, what bit rates (set by `-b`) you used?
2. Are the packets lost thought the test, or for example, they are lost mainly at the beginning of the test?
3. In general, low throughput compared to the network throughput should not cause any UDP packet loss, and throughput close or greater than the network throughput will cause high loss rate.  It seems that your network throughput is about 1Gbps.  What happens if you try to send 2Gbps (`-b 2G`)? What happens if you try 1Mbps?
